Sources/MessagingInApp/Gist/Managers/InlineMessageManager.swift

```diff
@@ -5,7 +5,7 @@ import UIKit
 // MARK: - InlineMessageManagerDelegate
 
 /// Callbacks specific to inline message events.
-public protocol InlineMessageManagerDelegate: AnyObject {
+protocol InlineMessageManagerDelegate: AnyObject {
```

Comment:
Removed `public` here as it required updates to view class but didn't look needed here. If it's needed, I can revert it to `public` and make required changes in view class as well.